examples/index.html

```diff
@@ -48,7 +48,6 @@ <h3 class="font-size-small">In Text:</h3><br/>
   <script>
     var blurredLocation = new BlurredLocation();
```

Comment:
There will be alot of options for Blurredlocation, don't you think we should put all of them in the documentation itself